celery/canvas.py

```diff
@@ -165,7 +160,6 @@ def __init__(self, task=None, args=None, kwargs=None, options=None,
                 options=dict(options or {}, **ex),
                 subtask_type=subtask_type,
                 immutable=immutable,
-                chord_size=None,
```

Comment:
do the changes related to chord_size going to break the current API?

Reply:
I don't think it should break anything and the tests are passing for me.  Is there a missing test case I should add to make sure the API is ok?

Reply:
If the integration tests will pass, everything is fine and this can be removed. Can you run them to make sure?

Reply:
Ran the integration tests with docker and I am getting the same results on this branch as I do on the master branch (1 failure that is unrelated to these changes).

![image](https://user-images.githubusercontent.com/1480930/109672180-c51cf780-7b42-11eb-9e09-a0cbe30237b1.png)